qabelbox/src/test/java/de/qabel/qabelbox/chat/interactor/ChatOverviewUseCaseTest.kt

```diff
@@ -0,0 +1,56 @@
+package de.qabel.qabelbox.chat.interactor
+
+import de.qabel.core.extensions.CoreTestCase
+import de.qabel.core.extensions.createContact
+import de.qabel.core.extensions.createIdentity
+import de.qabel.core.repository.entities.ChatDropMessage
+import de.qabel.core.repository.inmemory.InMemoryChatDropMessageRepository
+import de.qabel.core.repository.inmemory.InMemoryContactRepository
+import de.qabel.core.repository.inmemory.InMemoryIdentityRepository
+import de.qabel.qabelbox.chat.transformers.ChatMessageTransformer
+import org.hamcrest.Matchers.equalTo
+import org.hamcrest.Matchers.hasSize
+import org.junit.Assert.assertThat
+import org.junit.Before
+import org.junit.Test
+import java.util.*
+
+class ChatOverviewUseCaseTest : CoreTestCase {
+
+    val identity = createIdentity("Alice")
+    val contact = createContact("Bob")
+
+    val contactRepo = InMemoryContactRepository().apply {
+        save(contact, identity)
+    }
+    val identityRepo = InMemoryIdentityRepository().apply {
+        save(identity)
+    }
+    val chatRepo = InMemoryChatDropMessageRepository()
+    val useCase = MainChatOverviewUseCase(chatRepo, ChatMessageTransformer(identityRepo, contactRepo))
+
+    @Before
+    fun setUp() {
+        chatRepo.persist(createMsg("blubb"))
```

Comment:
You should reset the chatRepo in every test, otherwise you will get weird errors as soon as you create a second test method.